content/docs/guides/datadog.md

```diff
@@ -0,0 +1,114 @@
+- You have a Datadog account, your Datadog agent is set up, and you have the appropriate permissions to create an API key.
+
+## Steps to integrate Datadog with Neon
+
+1. In the Neon Console, navigate to the **Integrations** page in your Neon project.
+2. Locate the **Datadog** card and click **Add**.
+3. Enter your **Datadog API key**. You can generate or retrieve these from retrieve from [API Keys](https://app.datadoghq.com/organization-settings/api-keys) your Datadog organization.
+4. Select the **Datadog site** that best corresponds to your Neon project's region.
+5. Click **Confirm** to complete the integration.
+
+Once set up, Neon will start sending metrics to Datadog, and you can use these metrics to create custom dashboards and alerts in Datadog.
+
+## Example usage in Datadog
+
+Once integrated, you can create custom dashboards in Datadog by querying the metrics sent from Neon. Use Datadog's **Metrics Explorer** to search for metrics like `connection_counts`, `db_total_size`, and `host_cpu_seconds_total`. You can also set alerts based on threshold values for critical metrics.
+
+## Available metrics
+
+Neon makes the following metrics available for export to third parties; for now, availability is limited to the Datadog integration but will soon be expanded to other providers.
+
+All metrics include the following labels:
+
+- `project_id`
+- `endpoint_id`
+- `compute_id`
+- `job`
+
+Here's an example of the metric `db_total_size` with all labels:
+
+```json shouldWrap
+db_total_size{project_id="square-daffodil-12345678", endpoint_id="54321", compute_id="ep-shrill-blaze-b4hry7fg", job="sql-metrics"} = 10485760
+```
+
+<Admonition type="note">
+In Datadog, metric labels are referred to as `tags.` See [Getting Started with Tags](https://docs.datadoghq.com/getting_started/tagging/) in the Datadog Docs.
+</Admonition>
+
+| Name                                     | Job                  | Description                                                                                                                                              |
+| ---------------------------------------- | -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| connection_counts                        | sql-metrics          | Connection counts. The `state` label is reported as `state` from the `pg_stat_activity` Postgres view. Example values: `state="active"`, `state="idle"`. |
+| db_total_size                            | sql-metrics          | Size of all databases for the project, measured in bytes.                                                                                                |
+| lfc_approximate_working_set_size         | sql-metrics          | (DEPRECATED) Approximate working set size in pages of 8192 bytes.                                                                                        |
```

Comment:
@zaynetro should we bother including a deprecated metric or is it just included in exported metrics until the metric is removed?

Reply:
Let's not document it at least. I think we should not export it at all. I will create a task for us.

Reply:
removed.